README.md

```diff
@@ -93,10 +93,18 @@ All commands to be run from the root of this repo.
 
   To generate chart `README.md` files from the [template](ci/README.md.gotmpl):
 
+Linux
+
   ```console
   docker run --rm -v "$PWD:/helm-docs" jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl
   ```
 
+Windows (PowerShell)
+
+  ```console
+  docker run --rm -v ${PWD}:/helm-docs jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl
```

Comment:
What happens with the command if ´${PWD}` contains directory with spaces in the name? Will it work as expected?

Reply:
@Matthew-Beckett in your last PR, you had these vars shown like this `"${pwd}.Path:/helm-docs"`? I think what you have here is the same as linux as the curly braces will work there also?

Reply:
`PS > docker run --rm -v ${PWD}:/helm-docs jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl`

Works on Windows with both directory paths without spaces and those with. 

```console
time="2021-05-12T08:20:31Z" level=info msg="Generating README Documentation for chart /helm-docs/stable/weblate"
time="2021-05-12T08:20:31Z" level=info msg="Generating README Documentation for chart /helm-docs/stable/front stage"
```

@max-rocket-internet in regards to your suggestion about `${PWD}` with curly braces being compatible with both platforms, I have tested this and it seems to be the case, including directories that have spaces.

Should I update the PR to include the below command as the supported cross-platform command for helm-docs generation

```console
$ docker run --rm -v ${PWD}:/helm-docs jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl
```

Reply:
We still need the quotes for Mac/Linux though. But surely with quotes it works in Powershell also?

> Should I update the PR to include the below command

If we can keep the quotes and just add the curly braces then yes

Reply:
Done :)